test/login.spec.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-undef */
+import LoginScreen from './src/LoginScreen';
+import Base from './src/Base';
+import { startApp, stopApp } from './utilities';
+
+describe('Smoke Test', () => {
+  let app;
+  before(async () => {
+    app = await startApp();
+  });
+
+
+  after(async () => {
+    await stopApp(app);
+  });
+
+  it('should open', async () => {
+    Base.appIsOpen().should.eventually.equal(1);
+  });
+
+  it('should see if driver modal exists', async () => {
+   let check = await app.client.isExisting(LoginScreen.driverDismiss);  
+    if (check){
+        await app.client.click(LoginScreen.driverDismiss);
+    };
+});
+
+  it('should have all UI elements', async () => {
+    await LoginScreen.signUpLink.should.eventually.exist;
+    await LoginScreen.tidepoolLogo.should.eventually.exist;
+    await LoginScreen.uploaderLogo.getText()
+      .should.eventually.equal('Uploader');
+    await LoginScreen.forgotPasswordLink.should.eventually.exist;
+    await LoginScreen.supportLink.getText()
```

Comment:
I remember we had an issue for a while where the support link wasn't working. Can we also add tests here to check that the URLs don't 404?